crates/store/re_types/definitions/rerun/components/video_timestamp.fbs

```diff
@@ -4,9 +4,10 @@ namespace rerun.components;
 /// Timestamp inside a [archetypes.AssetVideo].
 struct VideoTimestamp (
   "attr.docs.unreleased",
+  "attr.rerun.experimental",
+  "attr.arrow.transparent",
   "attr.rust.derive": "Copy, PartialEq, Eq, Default",
-  "attr.rust.repr": "transparent",
-  "attr.rerun.experimental"
+  "attr.rust.repr": "transparent"
```

Comment:
You didn't like the lexicographical sorting? 😢

Reply:
hear me out 😄 : By putting experimental & docs at the top, they can be removed without messing with the non-trailing comma at the end!

everything else I don't have an excuse ;-)